data/crac-io/crac-io-json/src/main/java/com/powsybl/openrao/data/craciojson/serializers/NetworkActionSerializer.java

```diff
@@ -28,9 +27,13 @@ public void serialize(NetworkAction value, JsonGenerator gen, SerializerProvider
         gen.writeStringField(NAME, value.getName());
         gen.writeStringField(OPERATOR, value.getOperator());
         UsageRulesSerializer.serializeUsageRules(value, gen);
-        serializeElementaryActions(value, TOPOLOGICAL_ACTIONS, Arrays.asList(TerminalsConnectionAction.class, SwitchAction.class), gen);
-        serializeElementaryActions(value, PST_SETPOINTS, List.of(PhaseTapChangerTapPositionAction.class), gen);
-        serializeElementaryActions(value, INJECTION_SETPOINTS, Arrays.asList(GeneratorAction.class, LoadAction.class, DanglingLineAction.class, ShuntCompensatorPositionAction.class), gen);
+        serializeElementaryActions(value, TERMINALS_CONNECTION_ACTIONS, List.of(TerminalsConnectionAction.class), gen);
+        serializeElementaryActions(value, SWITCH_ACTIONS, List.of(SwitchAction.class), gen);
+        serializeElementaryActions(value, PHASETAPCHANGER_TAPPOSITION_ACTIONS, List.of(PhaseTapChangerTapPositionAction.class), gen);
+        serializeElementaryActions(value, GENERATOR_ACTIONS, List.of(GeneratorAction.class), gen);
+        serializeElementaryActions(value, LOAD_ACTIONS, List.of(LoadAction.class), gen);
+        serializeElementaryActions(value, DANGLINGLINE_ACTIONS, List.of(DanglingLineAction.class), gen);
+        serializeElementaryActions(value, SHUNTCOMPENSATOR_POSITION_ACTIONS, List.of(ShuntCompensatorPositionAction.class), gen);
```

Comment:
We could change the serialize method to not take a list of elementaryActionTypes but just one now.

Reply:
done